apps/frontend/src/app/5_pages/StakePage/components/StakingStatistics/StakingStatistics.tsx

```diff
@@ -25,6 +26,7 @@ import { GlobalStatistics } from '../../StakePage.utils';
 import { useGetStakingStatistics } from './hooks/useGetStakingStatistics';
 
 export const StakingStatistics = () => {
+  const chainId = useCurrentChain();
```

Comment:
Can't leave a comment there but line 34 needs to be updated to use this variable, currently it's pulling total staked SOV from RSK